python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3106,7 +3106,7 @@ def test_to_frame(pdf, gdf):
     gdf_new_name = gdf.x.to_frame(name=name)
     pdf_new_name = pdf.x.to_frame(name=name)
     assert_eq(gdf_new_name, pdf_new_name)
-    assert gdf_new_name.columns[0] is name
+    assert gdf_new_name.columns[0] == np.bool(name)
```

Comment:
Just to confirm, this was a change on the cuDF side?

Reply:
Ah gotcha, thanks.

Yeah looks like this is in the release notes: https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.5.0.html#other-enhancements

> Implemented a bool-dtype [Index](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Index.html#pandas.Index), passing a bool-dtype array-like to pd.Index will now retain bool dtype instead of casting to object ([GH45061](https://github.com/pandas-dev/pandas/issues/45061))

Where the `dtype` is `np.dtype(bool)`